script.js

```diff
@@ -0,0 +1,221 @@
+// User starts with either X or O
+// User clicks on a space, and that space shows an X or an O
+  // space displays the X or O, depending on which one was currently in effect
+  // turn switches to other player
+// Play continues until one of two things happens:
+    // No spaces remain
+    // A player, either X or O, has won
+// Reset button at the bottom of the page clears all entries
+
+var squareOne = $("#topRow .left");
+var squareTwo = $("#topRow .middle");
+var squareThree = $("#topRow .right");
+var squareFour = $("#middleRow .left");
+var squareFive = $("#middleRow .middle");
+var squareSix = $("#middleRow .right");
+var squareSeven = $("#bottomRow .left");
+var squareEight = $("#bottomRow .middle");
+var squareNine = $("#bottomRow .right");
+var resetButton = $("button");
+
+var playerTwo = false;
+
+function getWinner () {
+if (((squareOne.html() === "X") && (squareTwo.html() === "X") && (squareThree.html() === "X")) ||
```

Comment:
Here you've explicitly defined each win condition for both "o" and "x" giving a total of 16 combinations. Does the letter matter? or does it matter that it needs to be the same? In fact, a small quick refactoring would save you some characters per line. Something like this: `squareOne.html() === squareTwo.html() &&  squareOne.html() === squareThree.html()`.